Detectors/TOF/calibration/testWorkflow/TOFCalibCollectorWriterSpec.h

```diff
@@ -0,0 +1,126 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_CALIBRATION_TOFCALIB_COLLECTOR_WRITER_H
+#define O2_CALIBRATION_TOFCALIB_COLLECTOR_WRITER_H
+
+/// @file   TOFCalibCollectorWriterSpec.h
+/// @brief  Device to write to tree the information for TOF time slewing calibration.
+
+#include "TOFCalibration/TOFCalibCollector.h"
+#include "DataFormatsTOF/CalibInfoTOFshort.h"
+#include <TTree.h>
+#include <gsl/span>
+
+using namespace o2::framework;
+
+namespace o2
+{
+namespace calibration
+{
+
+class TOFCalibCollectorWriter : public o2::framework::Task
+{
+
+  using Geo = o2::tof::Geo;
+
+ public:
+  void createAndOpenFileAndTree()
+  {
+    TString filename = TString::Format("collTOF_%d.root", mCount);
+    LOG(DEBUG) << "opening file " << filename.Data();
+    mfileOut = new TFile(TString::Format("%s", filename.Data()), "RECREATE");
+    mOutputTree = new TTree("treeCollectedCalibInfo", "Tree with TOF calib info for Time Slewing");
```

Comment:
we are avoiding bare new/delete, could you make the mfileOut and mOutputTree ``std::unique_ptr<TFile> .. <TTree>`` resp., then assign them as 
````
mfileOut.reset( TFile::Open( ....) );
mOutputTree = std::make_unique<TTree>("treeCollectedCalibInfo", "Tree with TOF calib info for Time Slewing");
````
see below for cleaning

Reply:
Ok, I will test this way. 
What do you mean by "see below for cleaning"?

Reply:
done